README.md

```diff
@@ -28,6 +28,10 @@ bash alias dcp="docker-compose -f docker-compose-production.yml"
 
 #### Готовим код к работе
 ```bash
+mkdir fidals && cd fidals
```

Comment:
It's redundant string, I suppose. Let's every dev guy decide himself where to put project folders